AirCasting/Graph/Graph.swift

```diff
@@ -73,13 +73,13 @@ class UI_PollutionGraph: UIView {
         lineChartView.leftYAxisRenderer = renderer
     }
     
-    func updateWith(values: [Float]) {
+    func updateWith(thresholdValues: [Float]) {
         guard let renderer = renderer else { return }
```

Comment:
method does not describe that it is fail-able. Please consider [Fail-fast](https://www.google.com/search?client=safari&rls=en&q=fail+fast&ie=UTF-8&oe=UTF-8) rule. Make it failable and by walking upwards handle the optionally.

Reply:
I added it and learnt new thing, thanks ;)